app/views/coronavirus_local_restrictions/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <% content_for :breadcrumbs do %>
+        <%= render 'govuk_publishing_components/components/breadcrumbs', {
+          breadcrumbs: breadcrumbs,
+          collapse_on_mobile: true
+        } %>
+      <% end %>
+
+      <% content_for :title, "Collections" %>
+
+      <%= render "govuk_publishing_components/components/title", title: t("coronavirus_local_restrictions.lookup.title") %>
+
+      <%= render "govuk_publishing_components/components/inset_text", text: t('coronavirus_local_restrictions.lookup.inset_text') %>
+
+      <%= sanitize(t("coronavirus_local_restrictions.lookup.body_content")) %>
+
+      <%= render "shared/postcode-lookup" %>
```

Comment:
do we have the error message yet to pass into this?

Reply:
I think that will come when we add the POST route.

Reply:
oh yes - are we adding in the form in another PR?

Reply:
Yes. This ticket: https://trello.com/c/g3eVg2yg/833-display-the-lockdown-results

Reply:
yeah :) next task: https://trello.com/c/g3eVg2yg/833-display-the-lockdown-results